base/src/androidLibMain/kotlin/MainQueueDispatcher.kt

```diff
@@ -21,4 +21,5 @@ Copyright 2019 Splendo Consulting B.V. The Netherlands
 
 */
 
+@Deprecated("Use Dispatchers.Main")
```

Comment:
`@Deprecated("Use Dispatchers.Main", ReplaceWith("Dispatchers.Main"))`?

Reply:
This is just a copy/paste from another file already in master, I don't think the syntax you suggest is correct (AFAIK it needs imports), I will need to study that more